openmc/mgxs/mgxs.py

```diff
@@ -4809,64 +4849,953 @@ class Chi(MGXS):
         Whether or not the MGXS is merged from one or more other MGXS
     hdf5_key : str
         The key used to index multi-group cross sections in an HDF5 data store
+    mgxs : list of openmc.mgxs.MGXS
+        A list of MGXS to combine to compute this multi-group cross section
 
     """
 
     def __init__(self, domain=None, domain_type=None, groups=None,
```

Comment:
Yes, Python's Method Resolution Order [strategy](https://www.google.com/search?client=ubuntu&channel=fs&q=python+MRO&ie=utf-8&oe=utf-8) will select the `get_xs(...)` implementation from the `ScatterMatrixXS` parent class.